File: backend/src/main/java/iliev/yt/share/backend/user/dto/UserOutputDto.java
package iliev.yt.share.backend.user.dto;

import iliev.yt.share.backend.user.preferences.dto.UserPreferencesOutputDto;
import java.util.UUID;

public record UserOutputDto(
        UUID id,
        String firebaseUid,
        String email,
        String firstName,
        String lastName,
        UserPreferencesOutputDto userPreferences
) {
}

File: YTShare.Android/app/src/main/java/iliev/yt/share/mobile/data/remote/dto/UserOutputDto.kt
package iliev.yt.share.mobile.data.remote.dto

import kotlinx.serialization.Serializable

@Serializable
data class UserOutputDto(
    val id: String,
    val firebaseUid: String,
    val email: String,
    val firstName: String,
    val lastName: String
)
